src/ontology/edits/oeo-shared.omn

```diff
@@ -131,18 +131,87 @@ ObjectProperty: <http://purl.obolibrary.org/obo/IAO_0000136>
     
 ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000053>
 
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000118> "has characteristic"
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000020>
+    
     InverseOf: 
         OEO_00010121
     
     
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000056>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000002>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000003>
+    
+    
 ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000057>
 
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000003>
+    
+    
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000085>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000004>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000034>
+    
+    
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000086>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000002>
+    
     
 ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000087>
 
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000004>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000023>
+    
     
 ObjectProperty: <http://purl.obolibrary.org/obo/RO_0000091>
 
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000004>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000016>
+    
+    
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0001025>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000004>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000004>
+    
+    
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0002222>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000003>
+    
+    Range: 
+        <http://purl.obolibrary.org/obo/BFO_0000003>
+    
+    
+ObjectProperty: <http://purl.obolibrary.org/obo/RO_0002233>
+
+    Domain: 
+        <http://purl.obolibrary.org/obo/BFO_0000015>
+    
     
 ObjectProperty: <http://purl.obolibrary.org/obo/RO_0002234>
```

Comment:
Definition: _p has output c iff c is a participant in p, c is present at the end of p, and c is not present at the beginning of p._. It does not follow strictly from the definition, but following the parent object property (`has participant`), the domain should be `process` and the range should be `continuant`.